kubetest/main.go

```diff
@@ -459,7 +487,7 @@ func prepare() error {
 		return err
 	}
 
-	if err := os.MkdirAll("./_artifacts", 0777); err != nil { // Create artifacts
+	if err := os.MkdirAll(artifacts, 0777); err != nil { // Create artifacts
```

Comment:
I mention it coz I have been hit by this artifacts vs _artifacts before... if that's the designated behavior then I'm fine.